packages/web/src/tools/_string.scss

```diff
@@ -0,0 +1,12 @@
+@use 'sass:string';
```

Comment:
Credit maybe…? 🤷🏻‍♂️ 

```
// Author: Hugo Giraudel
```

https://www.google.com/search?q=hugo+giraudel+sass+string+replace

Reply:
It's Kitty now (https://kittygiraudel.com/about/). But thanks, added.